load-generator/README.md

```diff
@@ -15,8 +15,16 @@
     1. --dataFormat -d (eg, otlp, prometheus, xray)
 
 ### OTLP Metrics Load Test Sample Command,
+
+The following is a list of optional command line flags for configuration of OTLP metrics:
+* `--flushInterval, -f` : (default=1000)the metric collection export interval
+* `--metric_count, -m`: (default=1) the number of metrics that should be created for each metric type
+* `--datapointCount, -dp`: (default=1) the number of datapoints that should be created for each metric.
+* `--observationInterval, -o`: (default=1000) the interval at which metric observations/values are updated.
```

Comment:
Also needs units.

Reply:
Thanks. Updated read me accordingly. 
Also revised the statsdmetricemitter to fix the similar issue of load-generation, and grouped parameters in readme accordingly.